### DISCUSSION TITLE
How are multiple plugins handled?

### DISCUSSION THREAD

**Discussion by toddchap** [swiftbar/SwiftBar#389] [Q&A]
If I create multiple plugins is seems semi-random which one is displayed in the menu bar. What is the expected behavior? I can't seem to find it documented.

**Comment by toddchap** [comment]
After a while running I noticed that both my plugins appeared in the menu bar, but as soon as I edited one it went back to displaying only one. Now I'm wondering how long I will have to wait until they both appear again. Restarting SwiftBar didn't fix the issue.

**Comment by melonamin** [comment]
Hi @toddchap,  the answer to this question is very plugin specific, can you share the plugins you’re using?

SwiftBar will run\refresh multiple plugins at the same time and it definitely can display more than one plugin in the menu bar. If you don’t see anything in the menu bar it means one of the following:
- plugin is still refreshing, i.e. taking a long time to refresh
- plugin returned empty output - SwiftBar will not put anything in menu bar in this case
-

**Reply by toddchap** [reply]
Thanks for reaching out. I can't replicate the problem anymore. Thank you for offering to help.

**Comment by toddchap** [comment]
Can no longer reproduce.
